plugins/flowdock/samson_flowdock.gemspec

```diff
@@ -1,6 +1,8 @@
-Gem::Specification.new "samson_flowdock", "0.0.0" do |s|
-  s.summary = "Samson flowdock integration"
-  s.authors = ["Michael Grosser"]
-  s.email = "michael@grosser.it"
-  s.add_runtime_dependency "flowdock", "~> 0.3.1"
+Gem::Specification.new 'samson_flowdock', '0.0.0' do |s|
+  s.summary = 'Samson flowdock integration'
+  s.authors = ['Michael Grosser']
+  s.email = 'michael@grosser.it'
+  s.add_runtime_dependency 'flowdock', '~> 0.5.0'
+  s.files = Dir['{app,config,db,lib}/**/*']
+  s.test_files = Dir['test/**/*.rb']
```

Comment:
Looks like this will be deprecated soon, and it looks like we aren't using it directly:
https://github.com/rubygems/rubygems/issues/735